src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ReadClassifier.java

```diff
@@ -0,0 +1,124 @@
+        if ( !itr.hasPrevious() ) return false;
+
+        CigarElement lastEle = itr.previous();
+        if ( lastEle.getOperator() == CigarOperator.HARD_CLIP && itr.hasPrevious() ) {
+            lastEle = itr.previous();
+        }
+        return lastEle.getOperator() == CigarOperator.SOFT_CLIP &&
+                lastEle.getLength() >= MIN_SOFT_CLIP_LEN &&
+                isHighQualityRegion(read.getBaseQualities(), read.getLength() - lastEle.getLength());
+    }
+
+    public static boolean isHighQualityRegion( final byte[] quals, int idx ) {
+        for ( final int end = idx+MIN_SOFT_CLIP_LEN; idx != end; ++idx ) {
+            if ( quals[idx] < MIN_QUALITY ) return false;
+        }
+        return true;
+    }
+
+    public void checkBigIndel( final List<CigarElement> cigarElements,
+                               final GATKRead read,
+                               final List<BreakpointEvidence> evidenceList ) {
+        int locus = read.getStart();
+        for ( final CigarElement ele : cigarElements ) {
+            final CigarOperator op = ele.getOperator();
+            if ( ele.getLength() >= MIN_INDEL_LEN ) {
+                if ( op == CigarOperator.INSERTION ) {
+                    evidenceList.add(new BreakpointEvidence.LargeIndel(read, readMetadata, locus));
+                } else if ( op == CigarOperator.DELETION ) {
+                    evidenceList.add(new BreakpointEvidence.LargeIndel(read, readMetadata, locus+ele.getLength()/2));
+                }
+            }
+            if ( op.consumesReferenceBases() ) {
+                locus += ele.getLength();
+            }
+        }
+    }
+
+    public void checkDiscordantPair( final GATKRead read, final List<BreakpointEvidence> evidenceList ) {
+        if ( read.mateIsUnmapped() ) {
+            evidenceList.add(new BreakpointEvidence.MateUnmapped(read, readMetadata));
+        } else if ( !read.getContig().equals(read.getMateContig()) ) {
+            evidenceList.add(new BreakpointEvidence.InterContigPair(read, readMetadata));
+        } else if ( read.isReverseStrand() == read.mateIsReverseStrand() ) {
+            evidenceList.add(new BreakpointEvidence.SameStrandPair(read, readMetadata));
+        } else if ( (read.getStart() < read.getMateStart() && read.isReverseStrand()) ||
+                    (read.getStart() > read.getMateStart() && !read.isReverseStrand()) ) {
+            evidenceList.add(new BreakpointEvidence.OutiesPair(read, readMetadata));
+/*      } else {
+            final ReadMetadata.ReadGroupFragmentStatistics stats = readMetadata.getStatistics(read.getReadGroup());
+            final float zIshScore = (Math.abs(read.getFragmentLength()) - stats.getMedianFragmentSize()) /
+                                        stats.getMedianFragmentSizeVariance();
+
+            if ( zIshScore < MIN_ZISH_SCORE || (zIshScore > MAX_ZISH_SCORE && zIshScore < CRAZY_ZISH_SCORE) ) {
```

Comment:
For our future work to turn this back on, we might want to consider a) ignoring small zish scores, and b) weighting pieces of evidence more highly if they agree with other pieces of evidence w/r/t their mate mapping position for discordant pairs or their SA tag locations for split reads.